src/app/index.ts

```diff
@@ -331,17 +302,12 @@ module.exports = yo.extend({
     this.log(`      ${chalk.green('Congratulations!')} Your add-in has been created! Your next steps:\n`);
     this.log(`      1. Go the directory where your project was created:\n`);
     this.log(`         ${chalk.bold('cd ' + this._destinationRoot)}\n`);
-    this.log(`      2. Trust the Self-Signed Certificate for your local web server (if you haven't already done that).`);
-    this.log(`         For more information, visit https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md.\n`);
-    this.log(`      3. Start the local web server:\n`);
+    this.log(`      2. Start the local web server and sideload the add-in:\n`);
     this.log(`         ${chalk.bold('npm start')}\n`);
-    this.log(`      4. Sideload the add-in into your Office application:\n`);
-    this.log(`         ${chalk.bold('npm run sideload')}\n`);
-    this.log(`      5. Open the project in VS Code:\n`);
+    this.log(`      3. Open the project in VS Code:\n`);
     this.log(`         ${chalk.bold('code .')}\n`);
     this.log(`         For more information, visit http://code.visualstudio.com.\n`);
-    this.log(`      Please refer to resource.html in your project for additional information,`);
-    this.log(`      or visit our repo at: https://github.com/officeDev/generator-office.\n`);
+    this.log(`      Please visit https://docs.microsoft.com/en-us/office/dev/add-ins for more information about Office Add-ins.\n`);
```

Comment:
Please remove the locale (`en-us`) from the docs.microsoft.com URL.  URL should be this:

`https://docs.microsoft.com/office/dev/add-ins`

Reply:
fixed